.github/workflows/build.yml

```diff
@@ -17,4 +17,7 @@ jobs:
       - name: Lint
         run: |
           python -m py_compile contribute.py
-          flake8 contribute.py
\ No newline at end of file
+          flake8 contribute.py
```

Comment:
Please include the test file into the flake8 check.

Reply:
Okay. Should I use py_compile for test file as well?

Reply:
I guess it doesn't bring much value considering the test would be executed from the CI. You can also remove the py_compile step for the main script because of the same reason.

Reply:
Sure